doc/articles/get-started-rider.md

```diff
@@ -21,24 +21,26 @@ uid: Uno.GetStarted.Rider
 |                       | **Rider for Windows** | **Rider for Mac**  | **Rider for Linux** |
```

Comment:
@jeromelaban our templates for the issues some of them are already mentioning Rider, but only for Windows or macOS. We should add **Rider Linux** as well as an option right?
https://github.com/search?q=org%3Aunoplatform%20%22Rider%20Windows%22&type=code